script.js

```diff
@@ -0,0 +1,79 @@
+function startGame() {
+
+  for (var i = 1; i <= 9; i = i + 1) {
+    clearBox(i);
+  }
+
+  document.turn = "X";
+  if (Math.random() < 0.5) {
+    document.turn = "O";
+  }
+  document.winner = null;
+  setMessage(document.turn + " gets to start.");
+}
```

Comment:
I think this is fine here. But i would be careful of declaring variables in the document namespace because you may be overwriting something important in the future.